### PR TITLE
enhance category

### PR DESCRIPTION
**Pull request by AbdelrahmanMohyeldin**

### CODE REVIEW

**Comment by AbdelrahmanMohyeldin** [comment]
@fatmaalzharaa  changes done

File: retail-managment-system/rms-ui/src/app/pages/product/product.component.html [hunk 1]
```diff
@@ -86,32 +85,34 @@ <h3 class="panel-title fw-bold">
             </td>
           </ng-container>
 
-          <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+          <tr mat-header-row *matHeaderRowDef="displayedColumns;sticky:true;"></tr>
```
Comment (fatmaalzharaa) [inline retail-managment-system/rms-ui/src/app/pages/product/product.component.html:88]: Why do you need that "sticky:true" ?

Comment (AbdelrahmanMohyeldin) [reply retail-managment-system/rms-ui/src/app/pages/product/product.component.html:88]: To make the header row in table sticky while scrolling
chrome-capture-2023-1-15

Comment (fatmaalzharaa) [reply retail-managment-system/rms-ui/src/app/pages/product/product.component.html:88]: ok

**Review by fatmaalzharaa — APPROVED** [review]

**Pull request merged**